apps/dashboard/config/configuration_singleton.rb

```diff
@@ -43,7 +43,7 @@ def boolean_configs
       :per_cluster_dataroot         => false,
       :file_navigator               => false,
       :jobs_app_alpha               => false,
-      :files_app_remote_files       => false,
+      :remote_files_enabled         => false,
```

Comment:
@csc-swesters FYI the name of this config is changing.

Reply:
Thanks for the heads-up!